File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.js
function solution(l, r) {
    let arr = Array.from({length:r-l+1}, (v,i)=>i+l)
    arr = arr.filter(v=> v.toString().replace(/[0,5]/g, '') === '')
    return arr.length === 0 ? [-1] : arr;
}

File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int l, int r) {
    vector<int> answer;
    string num;
    for (int i = l; i <= r; i++) {
        bool b = true;
        num = to_string(i);
        for (auto l : num) {
            if (l != '5' && l != '0'){
                b = false;
            }
        }
        if (b)
            answer.push_back(i);
    }
    if(answer.empty())
        answer.push_back(-1);
    return answer;
}